sql/executor.go

```diff
@@ -206,8 +206,10 @@ func (e Executor) execStmt(stmt parser.Statement, params parameters, planMaker *
 						row.Values = append(row.Values, driver.Datum{
 							Payload: &driver.Datum_IntervalVal{IntervalVal: vt.Nanoseconds()},
 						})
+					case parser.DTuple:
+						return fmt.Errorf("unsupported result type: tuple, remove extra parenthesis around column %s", result.Columns[i])
```

Comment:
I don't want folks to see parser.DTuple, and I much prefer having error messages provide guidance on what to do. Should I say "you might want to remove the extra..."